schedule/__init__.py

```diff
@@ -322,47 +322,82 @@ def monday(self):
         self.start_day = 'monday'
         return self.weeks
 
+    @property
+    def mondays(self):
+        self.start_day = 'monday'
+        return self.weeks
+
     @property
     def tuesday(self):
         if self.interval != 1:
             raise IntervalError('Use tuesdays instead of tuesday')
         self.start_day = 'tuesday'
         return self.weeks
 
+    @property
+    def tuesdays(self):
+        self.start_day = 'tuesday'
+        return self
+
     @property
     def wednesday(self):
         if self.interval != 1:
             raise IntervalError('Use wednesdays instead of wednesday')
         self.start_day = 'wednesday'
         return self.weeks
 
+    @property
+    def wednesdays(self):
+        self.start_day = 'wednesday'
+        return self
+
     @property
     def thursday(self):
         if self.interval != 1:
             raise IntervalError('Use thursdays instead of thursday')
         self.start_day = 'thursday'
         return self.weeks
 
+    @property
+    def thursdays(self):
+        self.start_day = 'thursday'
+        return self
+
     @property
     def friday(self):
         if self.interval != 1:
             raise IntervalError('Use fridays instead of friday')
         self.start_day = 'friday'
         return self.weeks
 
+    @property
+    def fridays(self):
+        self.start_day = 'friday'
+        return self
+
     @property
     def saturday(self):
         if self.interval != 1:
             raise IntervalError('Use saturdays instead of saturday')
         self.start_day = 'saturday'
         return self.weeks
 
+    @property
+    def saturdays(self):
+        self.start_day = 'saturday'
+        return self
+
     @property
     def sunday(self):
         if self.interval != 1:
             raise IntervalError('Use sundays instead of sunday')
         self.start_day = 'sunday'
-        return self.weeks
```

Comment:
Wait, not only does this not do what you think it does, it also breaks existing functionality. This change would totally break the` .sunday` property...